Src/BootCamp.Chapter/Interact.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public static class Interact
```

Comment:
Class name should be a noun. This class is probably a LoggingDemo()?

Reply:
I actually forgot tor remove this class as I decided to get data from a list and not console input, but I would name it UserInteraction.

Reply:
I removed the class in [44d24bb](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/215/commits/44d24bb2b25bf887089cb41187d050e3aecc1d1d)